### ISSUE TITLE
Flip screen for CRT users?

### ISSUE THREAD

**Issue by Swainy101** [psomashekar/pr4m0d-MISTer-bin#30]
Great work guys, just wondering if you could add a flip option as I can't play the game upside down on my vertical CRT?

**Comment by psomashekar** [comment]
Is this not the rotate option already available in the core settings?

**Comment by Swainy101** [comment]
No, that’s for modern screens. There is a flip option in the dip switch
settings but that just mirrors the screen back to front.

**Comment by psomashekar** [comment]
Not sure then, but the dip switch inverts the screens. Not sure.

**Comment by zakk4223** [comment]
I just checked garegga against real hardware: The 'flip screen' dip does not work correctly in the core. On the PCB that option does exactly what Swainy is asking for (flips the screen 180 degrees so it works with monitors that rotate 'the wrong way'). In the core it seems to be some sort x-axis inversion(?), but it actually seems wrong. There's also graphical corruption/layers not clearing.
Checked Bakraid while I was writing this: same behavior

This was using the latest releases from the main mister repo.
